src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -119,9 +119,7 @@ case class WorkflowActor(workflow: WorkflowDescriptor,
    */
   private def startRunnableCalls(): State = {
     tryStartingRunnableCalls() match {
-      case Success(entries) =>
-        if (entries.nonEmpty) tryStartingRunnableCalls()
-        goto(WorkflowRunning)
+      case Success(entries) => if (entries.nonEmpty) startRunnableCalls() else goto(WorkflowRunning)
```

Comment:
I'll ask that both here and future cases where people say things like the above ("It shouldn't hold up GOTC! Let's fix it later!") we should have a ticket or something